dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/metadata/ConfigurableMetadataServiceExporter.java

```diff
@@ -113,7 +117,8 @@ private ProtocolConfig generateMetadataProtocol() {
 
         ProtocolConfig protocolConfig = new ProtocolConfig();
         protocolConfig.setName(specifiedProtocol);
-
+        String serialization = getSerialization(specifiedProtocol);
+        protocolConfig.setSerialization(serialization);
```

Comment:
BTW, please add some uts to verify this

Reply:
> BTW, please add some uts to verify this

We should do this, but need to establish a priority for the configuration reads